packages/ui-library/src/components/counter/index.ts

```diff
@@ -10,26 +10,26 @@ export class BlrCounter extends LitElement {
   static styles = [];
 
   @property() variant: CounterVariantType = 'neutral';
-  @property() current = 0;
-  @property() max = 0;
-  @property() size?: FormSizesType = 'md';
+  @property({ type: Number }) value = 0;
```

Comment:
Can it be more consistent because the type is defined for some and for some its missing.

Reply:
I only do that in these case where lit analyze is barking. Apart from that, this is a topic we still need to talk about anyway. From consistent I agree with you.